pkg/cloud/alibaba/assets/cloud-controller-manager-daemonset.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloud-config
+  namespace: kube-system
+data:
+  cloud-config.conf: |-
+    {
+        "Global": {
+            "routeTableIDs": ""
+        }
+    }
```

Comment:
Cloud config is typically handled by a separate component and not CCCMO, I wouldn't expect this to be here